docker/requirements/Checker.php

```
@@ -0,0 +1,644 @@
+        }
+        
+        return $this->compareByteSize($memoryLimit, $minimumLimit, '>=');
+    }
+
+    /**
+     * Check if command exists in PATH
+     */
+    public function checkCommandExists($command)
+    {
+        $whereIsCommand = (PHP_OS_FAMILY === 'Windows') ? 'where' : 'which';
+        exec("$whereIsCommand $command", $output, $returnCode);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$output'. (Unused Code Rules)

(UnusedLocalVariable)